label.go

```diff
@@ -15,26 +15,19 @@
 package rigging
 
 import (
-	"encoding/json"
-
 	"github.com/gravitational/trace"
+	v1 "k8s.io/api/core/v1"
+	v1meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
 )
 
-func NodesMatchingLabel(labelQuery string) (*NodeList, error) {
-	var nodes NodeList
-
-	cmd := KubeCommand("get", "nodes", "-l", labelQuery, "-o", "json")
-	out, err := cmd.Output()
-	if err != nil {
-		return nil, trace.Wrap(err)
-	}
-
-	err = json.Unmarshal(out, &nodes)
+func NodesMatchingLabel(client *kubernetes.Clientset, labelQuery string) (*v1.NodeList, error) {
+	nodes, err := client.CoreV1().Nodes().List(v1meta.ListOptions{LabelSelector: labelQuery})
```

Comment:
I think it's more common to use `metav1`.
Also, why do we need this method - it seems it does the work that a simple in-place:
```go
client.Nodes().List(metav1.ListOptions{LabelSelector: selector})
```
would do which is just as readable?